stories/controls/tab-bar.story.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { storiesOf, action } from '@kadira/storybook'
+import { TabBar as StaticTabBar } from 'src'
+import dynamicInput from '../dynamic-input'
+
+const TabBar = dynamicInput({
+})(StaticTabBar)
+
+storiesOf('TabBar', module)
+  .add('static', () => (
```

Comment:
Since we use `dynamicInput`, this is technically not static. I think we can remove it since it doesn't give us any extra insight into how the component works.